api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -672,16 +683,32 @@ public boolean isSkipAuthenticatorPresent(ProceedContext proceedContext) {
         return proceedContext.getSkipHref() != null;
     }
 
+    /** Begin transaction **/
     public AuthenticationResponse begin() {
         try {
-            return AuthenticationTransaction.create(client).asAuthenticationResponse();
+            return AuthenticationTransaction.create(client, Optional.empty()).asAuthenticationResponse();
+        } catch (ProcessingException e) {
+            return handleProcessingException(e);
+        } catch (IllegalArgumentException e) {
+            return handleIllegalArgumentException(e);
+        }
+    }
+
+    /** Begin transaction with activation token **/
+    public AuthenticationResponse begin(String activationToken) {
```

Comment:
I didn't see any new tests for this. Can we add one?

Reply:
https://github.com/okta/okta-idx-java/blob/okta-436460-add-activation-token/api/src/test/groovy/com/okta/idx/sdk/api/client/IDXAuthenticationWrapperTest.groovy#L363

Reply:
There aren't any new assertions there. If you removed the code that sends the query param, all the tests would still pass.

Reply:
Updated.